src/components/site/tableOfContents/table-of-contents.module.scss

```diff
@@ -0,0 +1,36 @@
+:global {
+  @include react-md-toolbars;
+  @include react-md-drawers;
+  @include react-md-lists;
+  @include react-md-buttons;
+  @include react-md-icons;
```

Comment:
probs not necessary if using our already styled button component (see comment on line 4)